CONTRIBUTING.md

```diff
@@ -31,9 +31,15 @@ Tests are handled with [pytest](https://docs.pytest.org/en/stable/index.html). T
 Once `h5grove` is installed in editable mode, run the tests with
 
 ```
-invoke test
+invoke test [--options]
```

Comment:
Just documenting the options I had previously added to the `invoke test` command.

Reply:
`[--options]` might be miss-leading since there is no `--options` option.
I would remove it and keep the list of available options below.